core/lib/zksync_core/src/sync_layer/tests.rs

```diff
@@ -589,180 +582,3 @@ async fn external_io_with_multiple_l1_batches() {
     assert_eq!(fictive_miniblock.timestamp, 2);
     assert_eq!(fictive_miniblock.l2_tx_count, 0);
 }
-
-#[tokio::test]
-async fn fetcher_basics() {
-    let pool = ConnectionPool::test_pool().await;
```

Comment:
Dumb question: Is it possible to adapt the removed tests for the new fetcher? IMO, they provide useful coverage for the fetcher logic.

Reply:
You mean to check for the specific sequence of actions? I would prefer to make the conversion FetchedBlock -> actions an implementation detail of ExternalIO instead.